### PR TITLE
Adjust TX Bolt protocol bytes

### PR DESCRIPTION
**Pull request by stanographer**
Found that the TX Bolt bytes sent from the machine are actually 0-3, not 6 as the code assumes.

### CODE REVIEW

**Comment by benoit-pierre** [comment]
Ah! My mistake actually, sorry... Why the timeout change?

File: plover/machine/txbolt.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 # byte represents. The next bits are set if the corresponding key was
 # pressed for the stroke.
 
-# 00XXXXXX 01XXXXXX 10XXXXXX 110XXXXX
+# 00XXXXXX 01XXXXXX 10XXXXXX 11XXXXXX
 #   HWPKTS   UE*OAR   GLBPRF    #ZDST
```
Comment (benoit-pierre) [inline plover/machine/txbolt.py]: I don't think this should be changed, as the line was correct: the last set has only 5 keys.

**Comment by stanographer** [comment]
Fixed the comment change. 100ms is still noticeable while writing at speed. 50ms seems much more "realtime" as I'm writing.

**Comment by benoit-pierre** [comment]
OK, does the last set detection help with regard to #258?

**Comment by morinted** [comment]
Using one of Stan's machines right now... no, it's still occurring. It's happening because the 0-byte is sent only after 10 seconds. However, the ending byte is now working properly.

**Comment by benoit-pierre** [comment]
But the timeout handling is supposed to finish the stroke anyway (that's why I asked why it was lowered, too low, and we could potentially split a stroke in 2).

**Comment by morinted** [comment]
Okay, then perhaps 100ms would be okay.

I've been playing with the machine... I think the reason the slowdown is happening is the same as I had way back when sleeping between key press outputs... instead of sleeping 0.1s, Plover starts sleeping for about 10 seconds between reads! I can see it in the console pretty clearly.

**Comment by benoit-pierre** [comment]
So it's a problem with the output code?

**Pull request closed without merging**